site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -13182,159 +13126,46 @@ private void setSelectedParticipantRoles(SessionState state) {
 
 	} // setSelectedParticipantRol3es
 
+	@Getter @AllArgsConstructor
 	public class MyIcon {
-		protected String m_name = null;
-
-		protected String m_url = null;
-
-		protected String m_skin = null;
-
-		public MyIcon(String name, String url, String skin) {
-			m_name = name;
-			m_url = url;
-			m_skin = skin;
-		}
-
-		public String getName() {
-			return m_name;
-		}
-
-		public String getUrl() {
-			return m_url;
-		}
-
-		public String getSkin() {
-			return m_skin;
-		}
+		private String name = null;
+		private String url = null;
+		private String skin = null;
 	}
 
 	// a utility class for working with ToolConfigurations and ToolRegistrations
 	// %%% convert featureList from IdAndText to Tool so getFeatures item.id =
 	// chosen-feature.id is a direct mapping of data
+	@Getter @EqualsAndHashCode
```

Comment:
I think generally this PR is tidying things up, but I don't see the value in removing the `hashCode` and `equals` which it appears someone has put thought into and the lombok versions do different things.

Would you be happy to drop this bit out of the PR?